config/samples/infrastructure_v1beta1_k8sinstallerconfig.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: K8sInstallerConfig
+metadata:
+  name: k8sinstallerconfig-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
Should we update these samples to add `BundleRepo` and `BundleType `?

Reply:
Are we going to use these samples?  
its empty for the [existing CR](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/config/samples/infrastructure_v1beta1_byomachine.yaml) thats why didn't added for this as well as.

Reply:
> Are we going to use these samples?

I don't think so, but this will be a good candidate for README, so users don't have to create CR from scratch, they can directly use it from the sample and customize it as per their needs.

> its empty for the [existing CR](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/main/config/samples/infrastructure_v1beta1_byomachine.yaml) thats why didn't added for this as well as.

Looks like the existing CR is filled with the default value `foo:bar` and hasn't been updated.

Reply:
K8sInstallerConfig CR will be part of `cluster.yaml` so we don't need this sample for now.